### PR TITLE
android: Add initial CIA installation

### PR DESCRIPTION
**Pull request by BreadFish64**
I'm not too familiar with async in the android UI but this seems to work without locking everything up. Deferring to @Schplee for an icon.
image
image

### CODE REVIEW

**Comment by FearlessTobi** [comment]
I hate the UI implications of this, but I have no better idea either :/

**Comment by BreadFish64** [comment]
Note to self: remove debug log

File: src/android/app/src/main/java/org/citra/citra_emu/ui/main/MainActivity.java [hunk 1]
```diff
@@ -171,6 +186,12 @@ protected void onActivityResult(int requestCode, int resultCode, Intent result)
                     mPresenter.onDirectorySelected(FileBrowserHelper.getSelectedDirectory(result));
                 }
                 break;
+                case MainPresenter.REQUEST_INSTALL_CIA:
+                    // If the user picked a file, as opposed to just backing out.
+                    if (resultCode == MainActivity.RESULT_OK) {
+                        NativeLibrary.InstallCIAS(FileBrowserHelper.getSelectedFiles(result));
+                    }
```
Comment (SachinVin) [inline src/android/app/src/main/java/org/citra/citra_emu/ui/main/MainActivity.java:193]: It is probably a good idea to refresh the game list once the install completes

**Comment by SachinVin** [comment]
(Feature request) Can we have progress bar ?

**Comment by Schplee** [comment]
Provided BreadFish with an icon via Discord

**Comment by BreadFish64** [comment]
@SachinVin a progress bar would be good but I don't think the added complexity is worth it right now. I'd rather focus on making sure this is stable and getting it merged ASAP.

**Pull request merged**